### PR TITLE
Mark cells as accessibility elemnts to avoid assertion failure in UI tests

### PR DESCRIPTION
**Pull request by benasher44**
This sets `isAccessibilityElement` to `true` for PhotoCells, which prevents a crash when trying to select them in UI tests.

### CODE REVIEW

File: Pod/Classes/Model/PhotoCollectionViewDataSource.swift [hunk 1]
```diff
@@ -64,6 +64,7 @@ final class PhotoCollectionViewDataSource : NSObject, UICollectionViewDataSource
         UIView.setAnimationsEnabled(false)
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: photoCellIdentifier, for: indexPath) as! PhotoCell
         cell.accessibilityIdentifier = "photo_cell_\(indexPath.item)"
+        cell.isAccessibilityElement = true
```
Comment (ronaldsmartin) [inline Pod/Classes/Model/PhotoCollectionViewDataSource.swift:67]: Curious as to why this needs to be specified when the docs say it's the default for UIKit controls?
https://developer.apple.com/documentation/objectivec/nsobject/1615141-isaccessibilityelement

Either way, sweet fix! 😄

**Review by ronaldsmartin — APPROVED** [review]
👏 can you also help me understand how you knew that this was how to fix this?

File: Pod/Classes/Model/PhotoCollectionViewDataSource.swift (hunk 1 above)
Comment (ronaldsmartin) [reply Pod/Classes/Model/PhotoCollectionViewDataSource.swift:67]: Oh, is it because a cell isn't a control?

Comment (asmallteapot) [reply Pod/Classes/Model/PhotoCollectionViewDataSource.swift:67]: That’s right, cells don’t subclass UIControl.

Comment (benasher44) [reply Pod/Classes/Model/PhotoCollectionViewDataSource.swift:67]: I found it through some SO-diving.

**Pull request merged**